.github/workflows/deploy.yml

```
@@ -0,0 +1,105 @@
+
+      - name: EC2 배포 실행
+        uses: appleboy/ssh-action@v0.1.10
+        with:
+          host: ${{ secrets.EC2_HOST }}
+          username: ubuntu
+          key: ${{ secrets.EC2_PRIVATE_KEY }}
+          script: |
+            # 배포 스크립트 실행 권한 부여 및 실행
+            sudo chmod +x /home/ubuntu/deploy.sh
+            /home/ubuntu/deploy.sh
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)